components/cascader/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -1518,11 +1512,8 @@ exports[`Cascader should render not found content 1`] = `
       motionEnter={false}
       motionLeave={false}
       motionName="slide-up"
-      onAppearStart={[Function]}
```

Comment:
You should't update this, please run `npm test -- -u` after reinstall `node_modules`.